db/migrate/20210224190247_migrate_shipping_taxes.rb

```diff
@@ -0,0 +1,83 @@
+class MigrateShippingTaxes < ActiveRecord::Migration
```

Comment:
How do we verify this giant?

Ah, now I see that you added a spec. Very cool. If you do that in the same commit next time then I don't have to wonder if this is covered.